content/applications/hr/employees/new_employee.rst

```diff
@@ -231,36 +249,38 @@ This tab provides various fields for different information, depending on the cou
 located. Different fields are configured for different locations, however some sections appear
 regardless.
 
-- :guilabel:`Status`: If applicable, select a :guilabel:`Related User`, :guilabel:`Job Position`,
-  and :guilabel:`Language` with the drop-down menus. Type in the :guilabel:`Registration Number of
-  the Employee`, and the :guilabel:`NIF Country Code` if available.
-- :guilabel:`Fleet`: If the employee has access to a company car, enter the :guilabel:`Mobility
-  Card` information here.
-- :guilabel:`Timesheets`: Enter the employee's cost per hour in a $XX.XX format. This is factored in
-  when the employee is working at a work center. This value affects the employee's pay, and may also
-  affect manufacturing costs for a product, if the value of the manufactured product is not a fixed
-  amount.
-- :guilabel:`Attendance`: This section may only appear for Belgian companies. Enter the
-  :guilabel:`INSZ or BIS` number if applicable. :guilabel:`Badge ID` and a :guilabel:`PIN Code` can
-  be entered here, if the employee needs/has one. Click :guilabel:`Generate` next to the
-  :guilabel:`Badge ID` to create a badge ID.
+- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a :guilabel:`Related
+  User`, with the drop-down menus.
+- :guilabel:`Payroll`: If applicable, enter the :guilabel:`Registration Number of the Employee` in
+  this section. The :guilabel:`Attestation (N-1)` and :guilabel:`Attestation (N)` sections appear
+  ony for Belgian companies, and will not be visible for other locations. These sections log the
+  days that will be paid to the new employee. Enter any :guilabel:`Amount to recover`,
+  :guilabel:`Number of days`, and :guilabel:`Recovered Amount` of simple holiday pay, for both N and
+  N-1 categories. For the double holiday pay to recover section, click :guilabel:`Add a line`, and
```

Comment:
I can't see this in Runbot to verify, but it seems like the "double holiday pay to recover" section should have a guilabel or some other kind of formatting on it?

Reply:
Yes- I added the full name of the line (it is very long!) for both sections.